src/Core/wwwroot/js/dialog-utils.js

```diff
@@ -0,0 +1,7 @@
+export function getActiveElement() {
+    return document.activeElement
+}
+
+export function focusElement(element) {
```

Comment:
Yo avoid side effect in some situations, can you test if the element is defined?
```js
if (element) {
  element.focus();
}
```

Reply:
Done!